specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json

```diff
@@ -100,6 +100,80 @@
                 }
             }
         },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/moveSubscriptions": {
+            "post": {
+                "x-ms-examples": {
+                    "Move user subscriptions from the Default Provider to a Delegated Provider.": {
+                        "$ref": "./examples/Subscriptions/MoveSubscriptionsToDelegatedProvider.json"
+                    },
+                    "Move user subscriptions from a Delegated Provider to the Default Provider.": {
+                        "$ref": "./examples/Subscriptions/MoveSubscriptionsToDefaultProvider.json"
+                    }
+                },
+                "description": "Move subscriptions between delegated provider offers.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "MoveSubscriptions",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/MoveSubscriptionsParameter"
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "NOCONTENT"
+                    },
+                    "202": {
+                        "description": "ACCEPTED"
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/validateMoveSubscriptions": {
+            "post": {
+                "x-ms-examples": {
+                    "Validate that user subscriptions can be moved from the Default Provider to a Delegated Provider.": {
+                        "$ref": "./examples/Subscriptions/ValidateMoveSubscriptionsToDelegatedProvider.json"
+                    },
+                    "Validate that user subscriptions can be moved from a Delegated Provider to the Default Provider.": {
+                        "$ref": "./examples/Subscriptions/ValidateMoveSubscriptionsToDefaultProvider.json"
+                    }
+                },
+                "description": "Validate that user subscriptions can be moved between delegated provider offers.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "ValidateMoveSubscriptions",
```

Comment:
use `Subscriptions_ValidateMove` or `Subscriptions_ValidateMoveSubscriptions`, just make sure it is consistent with other operation ids